react/Paywall/Paywall.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+
+import Icon from '../Icon'
+
+import { IllustrationDialog } from '../CozyDialogs'
+import Button from '../Button'
```

Comment:
Ne devrait-on pas avoir ../Buttons ici ?

Reply:
For sure 😕 `/legacy` will help